common/json_compare.go

```diff
@@ -0,0 +1,111 @@
+package common
```

Comment:
Since this file has other json functions, like sorting, which may be handy elsewhere, lets name the file to something more generic for json functions, like `json.go` or `json_helpers.go`, and we can keep adding to it.

Reply:
Sure. Will provide a generic name as suggested above.